app/components/channel_list/categories/body/category_body.tsx

```diff
@@ -40,6 +44,16 @@ const CategoryBody = ({category, categoryChannels, channels, myChannels}: Props)
         <FlatList
             data={data}
             renderItem={ChannelItem}
+            keyExtractor={extractKey}
+            removeClippedSubviews={true}
+            initialNumToRender={20}
+            windowSize={15}
+            updateCellsBatchingPeriod={10}
+            maxToRenderPerBatch={10}
+            getItemLayout={(d, index) => (
+                {length: 40, offset: 40 * index, index}
+            )}
+            viewabilityConfig={List.VISIBILITY_CONFIG_DEFAULTS}
```

Comment:
why do we care about the visibility of items in this list?

Reply:
Whoops - for this PR we don't; it's more optimisations for when collapsing is happening.